Shopstagram/css/components/image-square/_image-square.scss

```diff
@@ -0,0 +1,33 @@
+.image-square {
+    display: flex;
+    align-items: center;
+    justify-self: center;
+    width: 56px;
+    height: 56px;
+    border-radius: 100%;
+    overflow: hidden;
+
+    &--small {
+        width: 32px;
+        height: 32px;
+    }
+
+    &--tiny {
+        width: 24px;
+        height: 24px;
+    }
+}
+
+.image-square {
```

Comment:
po co to jest 2 razy ? masz to juz w 1 linii

Reply:
fixed